### ISSUE TITLE
Using large geotiff with Deck.gl

### ISSUE THREAD

**Issue by jurajhrib** [gisat-panther/deck.gl-geotiff#38]
Hey guys. First of all, thanks for your work. This is the first library I found that implements geotiff layers in Deck.gl.

I am building a small react GIS project for visualisation data in archaeology. I have a big geotiff of Czech DMR generated by my friend from open data. Originally, is used in QGIS project by combining two big geotiffs (8bit as base and 16bit for some shading). They are both around 7GB in size. 

I am not very skilled in GIS stuff and using geotiffs, but I would like to have this as layer in my app. I tried to convert the 8bit file into COG and change coordinate system from Krovak to WGS84. The file grew to around 11GB. I uploaded it to S3 and tried to load it via QGIS and COG explorer. It somewhat works (the raster has some small holes, but I presume it can be fixed by using different settings in conversion).

When I tried to load it via geotiff layer in Deck.gl, it did not load. I wanted to ask, whether there is some performance limit for loading COGs. 

If you want to look at the COG, this is the file in S3 bucket. I will later rename the file, because I am afraid of bot/scraper traffic.

`acolyte-cog.s3.eu-central-1.amazonaws.com/cog/dmr.cog.8bit.tif`

Basically, I would like to know, whether it is possible to do this using Deck.GL, or it's too heavy lift. I know these are large files, but I suppose that COG should handle this. It works when I load it into QGIS.

This is how it is visualised in QGIS project:
<img width="1370" alt="Snímek obrazovky 2023-09-23 v 18 09 06" src="https://github.com/gisat-panther/deck.gl-geotiff/assets/373788/43958e6a-a705-47ae-b8f3-5355da3c76b3">

**Comment by jurajhrib** [comment]
@vdubr thank you for your reply. I will try to create a different COG that would accommodate your requirements. Maybe I should first try to create a DMR of smaller area, so I can test which version will work. Because working with data of whole Czech republic takes a lot of time, when you make some transformations.
